packages/devtools_app/benchmark/README.md

```diff
@@ -23,6 +23,16 @@ To run the performance benchmark tests locally, run:
 dart run benchmark/scripts/run_benchmarks.dart
 ```
 
+Several arguments are available from the `run_benchmarks.dart` script. Using
+these arguments, you can do things like:
+* compute the average of multiple benchmark runs
+* compute a delta against a prior benchmark run
+* save the benchmark results to a file
+* run the benchmarks in the browser
+* run the benchmarks with the `dart2wasm` compiler
+
+Run `dart run benchmark/scripts/run_benchmarks.dart -h` to see a full list.
```

Comment:
to see a full list -> to see details


As previous text does not give any list of parameters, it just explains what is possible